spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/autoconfig/TraceAutoConfiguration.java

```diff
@@ -43,6 +37,11 @@
 import org.springframework.cloud.sleuth.SpanNamer;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import zipkin2.Span;
+import zipkin2.reporter.Reporter;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
I'll revert this, IntelliJ organize imports apparently didn't like the old order